src/Test.Utilities/MinimalImplementations/ASPNetCoreApis.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Test.Utilities.MinimalImplementations
+{
+    public static class ASPNetCoreApis
+    {
```

Comment:
Could also make other analyzer test cases that work with ASP.NET Core also use this.